testsuite/MDAnalysisTests/coordinates/test_chainreader.py

```diff
@@ -345,7 +348,17 @@ def test_easy_trigger_warning(self, tmpdir):
         sequences = ([0, 1, 2, 3], [2, 3, 4, 5], [4, 5, 6, 7])
         utop, fnames = build_trajectories(folder, sequences=sequences,)
         with no_warning(UserWarning):
-            mda.Universe(utop._topology, fnames, continuous=True)
+            with warnings.catch_warnings():
+                # for windows Python 3.10 ignore:
+                # ImportWarning('_SixMetaPathImporter.find_spec() not found
+                # TODO: remove when we no longer have a dependency
```

Comment:
Seems sensible to me.

Could this be raised as an issue so we remember to deal with this in the future?

Reply:
see gh-3485 for the reminder; also, note that the `xfail` has been restored from previously being commented out--so this test has clearly been a pain point in the past (we could also be more granular with the `xfail`, since it only occurs for one of the parameters I think, but since the `xfail` was previously used more broadly I've simply restored it for Windows).